RELEASE.md

```diff
@@ -48,6 +49,58 @@ Check issues at https://github.com/Icinga/icinga2
 For minor versions you need to manually backports any and all commits from the
 master branch which should be part of this release.
 
+### Windows Dependencies <a id="windows-dependencies"></a>
+
+In contrast to Linux, the Windows dependencies (at least Boost and OpenSSL)
+are neither updated independently of Icinga itself by instance administrators,
+nor their latest versions available for the OS are fetched at package build time.
+
+To ensure the upcoming Icinga release ships the latest (i.e. most secure) dependencies on Windows:
+
+#### Update packages.icinga.com
+
+Add the latest Boost and OpenSSL versions to
+https://packages.icinga.com/windows/dependencies/ like this:
+
+```
+localhost:~$ ssh icinga-aptly.icinga.netways.de
+icinga-aptly:~$ sudo -i
+icinga-aptly:~# cd /var/www/html/aptly/public/windows/dependencies
+icinga-aptly:dependencies# wget https://master.dl.sourceforge.net/project/boost/boost-binaries/1.76.0/boost_1_76_0-msvc-14.2-64.exe
+icinga-aptly:dependencies# wget https://master.dl.sourceforge.net/project/boost/boost-binaries/1.76.0/boost_1_76_0-msvc-14.2-32.exe
+icinga-aptly:dependencies# wget https://slproweb.com/download/Win64OpenSSL-1_1_1k.exe
+icinga-aptly:dependencies# wget https://slproweb.com/download/Win32OpenSSL-1_1_1k.exe
+```
+
+#### Ensure Compatibility
+
+Preferably on a fresh Windows VM (not to accidentally build Icinga
+with old dependency versions) setup a dev environment using the new dependency versions:
+
+1. Download [doc/win-dev.ps1](doc/win-dev.ps1)
+2. Edit your local copy, adjust the dependency versions
+3. ensure there are 35 GB free space on C:
+4. run the following in an administrative Powershell:
```

Comment:
Should start with an upper-case letter for consistency.

Reply:
## Blocked by

* #8851